apps/hash-frontend/src/pages/[shortname].page.tsx

```diff
@@ -193,13 +193,20 @@ const ProfilePage: NextPageWithLayout = () => {
               metadata.recordId.baseUrl === tab.entityTypeBaseUrl,
           );
 
-          const entityTypeAndDescendants =
-            entityType && entitiesSubgraph
-              ? getEntityTypeAndDescendantsById(
-                  entitiesSubgraph,
-                  entityType.schema.$id,
-                )
-              : [];
+          let entityTypeAndDescendants = entityType ? [entityType] : [];
+          try {
+            entityTypeAndDescendants =
+              entityType && entitiesSubgraph
+                ? getEntityTypeAndDescendantsById(
+                    entitiesSubgraph,
+                    entityType.schema.$id,
+                  )
+                : [];
```

Comment:
I think we still want to initialise with the available `entityType` if we can so the title is displayed

```suggestion
            entityTypeAndDescendants =
              entityType && entitiesSubgraph
                ? getEntityTypeAndDescendantsById(
                    entitiesSubgraph,
                    entityType.schema.$id,
                  )
                : entityType
                ? [entityType]
                : [];
```

Reply:
Addressed in https://github.com/hashintel/hash/pull/3572/commits/45f0e28ff4ca3c7c80ced64291bd02dbf27eea79